elixir/apps/api/lib/api/controllers/gateway_json.ex

```diff
@@ -0,0 +1,44 @@
+defmodule API.GatewayJSON do
+  alias Domain.Gateways
+  alias Domain.Repo.Paginator.Metadata
+
+  @doc """
+  Renders a list of Gateways.
+  """
+  def index(%{gateways: gateways, metadata: metadata}) do
+    %{data: for(gateway <- gateways, do: data(gateway))}
+    |> Map.put(:metadata, metadata(metadata))
+  end
+
+  @doc """
+  Render a single Gateway
+  """
+  def show(%{gateway: gateway}) do
+    %{data: data(gateway)}
+  end
+
+  @doc """
+  Render a Gateway Token
+  """
+  def token(%{gateway_token: token}) do
```

Comment:
This also should be in `gateway_group_json.ex`.

Reply:
👍 Yeah, I've already moved it in my local branch after @jamilbk suggested moving the token creation under `gateway_groups` rather than `gateways`.  I'll double check that I've removed it from this file though.